e2e-manifests/labels-with-all-legal-characters.yml

```diff
@@ -0,0 +1,59 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: labels-with-all-legal-characters
+  labels:
+    illuminatio-e2e: labels-with-all-legal-characters
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test
+  namespace: labels-with-all-legal-characters
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      test.io/test-123_XYZ: test_456-123.ABC
+  template:
+    metadata:
+      labels:
+        test.io/test-123_XYZ: test_456-123.ABC
+    spec:
+      containers:
+      - image: nginx
+        imagePullPolicy: Always
+        name: test
+        ports:
+        - containerPort: 80
+          protocol: TCP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: test
+  namespace: labels-with-all-legal-characters
+spec:
+  type: ClusterIP
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 80
+  selector:
+      test.io/test-123_XYZ: test_456-123.ABC
+---
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: api-allow
+  namespace: labels-with-all-legal-characters
+spec:
+  podSelector:
+    matchLabels:
+      test.io/test-123_XYZ: test_456-123.ABC
+  ingress:
+  - from:
+      - podSelector:
```

Comment:
You mixed up 2 spaces and 4 spaces. I would prefer consistently to use them consistently.